hc/front/tests/test_add_pushbover.py

```diff
@@ -37,4 +38,13 @@ def test_it_validates_nonce(self):
         r = self.client.get("/integrations/add_pushover/?%s" % params)
         assert r.status_code == 403
 
-    ### Test that pushover validates priority
+        ### Test that pushover validates priority
+    def test_it_validates_priority(self):
+        self.client.login(username="alice@example.org", password="password")
+
+        session = self.client.session
+        session["po_nonce"] = "n"
+        session.save()
+
+        params = "pushover_user_key=a&nonce=INVALID&prio=123"
+        r = self.client.get("/integrations/add_pushover/?%s" % params)
```

Comment:
thank you for the feedback, i now have a clear understanding of what I need to test.